stacks/core/dev/stage-1.yml

```diff
@@ -41,6 +41,20 @@ services:
         - traefik.port=8183
 
   # KNoT Fog Core
+  storage:
+    image: cesarbr/knot-cloud-storage-go:dev
+    env_file: './env.d/knot-cloud-storage.env'
+    volumes:
+      - ../knot-cloud-storage:/usr/src/app
+    depends_on:
+      - mongo
+    deploy:
+      replicas: 1
+      labels:
+        - traefik.enable=true
+        - traefik.frontend.rule=HostRegexp:data,data.{domain:[a-zA-Z0-9.]+}
+        - traefik.port=8181
```

Comment:
Is the `storage` exporting this port?

Reply:
Not in the current release. But I create a PR to change this